### PR TITLE
`CuboidTag` and `EllipsoidTag` examples

### PR DESCRIPTION
**Pull request by BreadcrumbIsTaken**
#### Adds examples to `CuboidTag` and `EllipsoidTag` tags and mechs

Notes:
- The tag `<CuboitTag.set[].at[]>` and the mechanism `set` was ignored due to me not being really sure what they did and how to use them lol. I might do more testing later and add examples in a future PR.

### CODE REVIEW

File: plugin/src/main/java/com/denizenscript/denizen/objects/CuboidTag.java [hunk 1]
```diff
@@ -890,6 +915,11 @@ public static void register() {
         // @mechanism CuboidTag.add_member
         // @description
         // Returns a modified copy of this cuboid, with the input cuboid(s) added at the end.
+        // @example
+        // # Adds "my_second_cuboid" as a member of "my_cuboid".
```
Comment (mcmonkey4eva) [inline plugin/src/main/java/com/denizenscript/denizen/objects/CuboidTag.java]: Not quite. It creates a *new* cuboid, named `my_third_cuboid`, which contains both `my_cuboid` and `my_second_cuboid` as members

File: plugin/src/main/java/com/denizenscript/denizen/objects/CuboidTag.java [hunk 2]
```diff
@@ -1238,6 +1317,11 @@ public static void register() {
         // and the output max will contain the old min values.
         // Note that this is equivalent to constructing a cuboid with the input value and the original cuboids min value.
         // Not valid for multi-member CuboidTags.
+        // @example
+        // # Notes a new cuboid with the player's location as the new maximum location of the cuboid.
+        // # For example: if "my_cuboid" spans from 10,10,10 (max) to 5,5,5 (min) and the player had a location of 15,10,15,
+        // # then "my_new_cuboid" will have a minimum location of 15,10,15.
```
Comment (mcmonkey4eva) [inline plugin/src/main/java/com/denizenscript/denizen/objects/CuboidTag.java]: Typo, min vs. max
Also for clarity's sake should probably rephrase this one and the above to more like `will span from 15,10,15 (max) to 5,5,5 (min)` (ie match the original phrasing in a way that emphasizes what changed and what hasn't)

File: plugin/src/main/java/com/denizenscript/denizen/objects/CuboidTag.java [hunk 3]
```diff
@@ -1257,6 +1341,14 @@ public static void register() {
...
         // Note that you can also specify a single number to expand all coordinates by the same amount (equivalent to specifying a location that is that value on X, Y, and Z).
         // Not valid for multi-member CuboidTags.
+        // @example
+        // # If "my_cuboid" spans from 10,10,10 to 5,5,5 and gets expanded by 15 (15,15,15),
+        // # "my_expanded_cuboid" will span -10,-10,-10 to 25,25,25
+        // - note <cuboid[my_cuboid].expand[15]> as:my_expanded_cuboid
+        // @example
+        // # If "my_cuboid" spans from 10,10,10 to 5,5,5 and gets expanded by 15,15,15,
+        // # "my_expanded_cuboid" will span -10,-10,-10 to 25,25,25
+        // - note <cuboid[my_cuboid].expand[15,15,15]> as:my_expanded_cuboid
```
Comment (mcmonkey4eva) [inline plugin/src/main/java/com/denizenscript/denizen/objects/CuboidTag.java]: Should have different numbers in the 3 coordinates for this example, to emphasize what makes it different from the other.

File: plugin/src/main/java/com/denizenscript/denizen/objects/CuboidTag.java [hunk 4]
```diff
@@ -1332,6 +1430,12 @@ public static void register() {
         // Inverted by <@link tag CuboidTag.expand_one_side>
         // Not valid for multi-member CuboidTags.
         // If you shrink past the limits of the cuboid's size, the cuboid will flip and start expanding the opposite direction.
+        // @example
+        // # Shrinks the high value of the cuboid "my_cuboid" by 15 (15,15,15) on one side and notes it as "my_expanded_cuboid".
+        // - note <cuboid[my_cuboid].expand_one_side[15]> as:my_expanded_cuboid
```
Comment (mcmonkey4eva) [inline plugin/src/main/java/com/denizenscript/denizen/objects/CuboidTag.java]: copypasta typo

File: plugin/src/main/java/com/denizenscript/denizen/objects/CuboidTag.java [hunk 5]
```diff
@@ -1408,6 +1515,9 @@ public static void register() {
         // @returns ListTag(ChunkTag)
         // @description
         // Gets a list of all chunks partially or entirely within the CuboidTag.
+        // @example
+        // # Loads the chunks that are within the cuboid "my_cuboid", even if they are partially within the cuboid.
```
Comment (mcmonkey4eva) [inline plugin/src/main/java/com/denizenscript/denizen/objects/CuboidTag.java:1529]: The point of examples should be to, well, *give an example*! This example meta is just reiterating what the description is, without adding any real example data. Should have some numbers or something that makes clear what it means to be 'partially' within vs fully

File: plugin/src/main/java/com/denizenscript/denizen/objects/CuboidTag.java [hunk 6]
```diff
@@ -1547,6 +1661,12 @@ public void adjust(Mechanism mechanism) {
         // <CuboidTag.get>
         // <CuboidTag.add_member[<cuboid>]>
         // <CuboidTag.add_member[<cuboid>].at[<#>]>
+        // @example
+        // # Adds "my_second_cuboid" as a member to "my_cuboid".
+        // - adjust <cuboid[my_cuboid]> add_member:my_second_cuboid
```
Comment (mcmonkey4eva) [inline plugin/src/main/java/com/denizenscript/denizen/objects/CuboidTag.java:1680]: For all these member-related examples: again, add example data. What does the cuboid look like before/after this operation?

**Comment by BreadcrumbIsTaken** [comment]
Fixes pushed. Also changed a handful of `EllipsoidTag` examples to address reviews made for `CuboidTag` as well.

**Pull request merged**